### DISCUSSION TITLE
Hint users to put large logs in a Markdown <summary> and <details> element?

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#9241] [Suggest an Idea]
**EDIT:** This discussion is resolved with the merge of PR #9242.

---

## Problem

I notice that we have some users that open discussions/issues and copy/paste large logs into the top post.
The problem with doing this is that it pushes the replies and further discussion **way** down the page, and so leads to a lot of scrolling.

## Trick

I found a nice trick:

```markdown
<details><summary>Click me to see logs</summary>
<p>

PUT 3 BACKTICKS HERE
Your long log goes here.
PUT 3 BACKTICKS HERE

</p>
</details>
```

Live example of the trick:

<details><summary>Click me to see logs</summary>
<p>

```
Your long log goes here.
```

</p>
</details>

## Proposal

Do you want a PR from me to put this trick somewhere appropriate in our documentation?
That way we could put a link to this cool trick in our PR/Issue templates.

**Comment by rarkins** [comment]
One possibility would be a new label like `logs-required` and then that inserts a commend that includes the hint?

**Reply by HonkingGoose** [reply]
I like that idea, but I'll need to do some testing first to see if the label action accepts the backticks and `<details>` and `<summary>` elements in the `.yml` configuration file, because it might not work properly.

**Reply by rarkins** [reply]
Maybe we can add a doc for it and then link to it from the autogenerated comment if we need to?

**Reply by HonkingGoose** [reply]
I did some testing, and I cannot get the bot to format/render the Markdown "log hint" properly... 😢 
I tried 3 variations of the markdown hint, and none rendered as intended.

Putting a simple Markdown link in a comment works properly. 😄

**Reply by HonkingGoose** [reply]
Actually I got it working properly now! 🚀 

The solution was to remove the three backticks completely, and instead use 4 spaces to indent the code block.

**Comment by HonkingGoose** [comment]
@rarkins We still need a `logs:problem` label though, for any any of this to work. Can you create the `logs:problem` label on this repository? I don't have the rights to create a label. 😉

**Comment by HonkingGoose** [comment]
Closing this discussion as resolved, we now use Issue Forms which have the correct syntax pre-filled for our users. 😄 

> **Note**
> The new GitHub slash command can insert the correct syntax for the `<summary>` and `<details` element. Just enter `/details` into the issue/discussion/comment field. Read the [GitHub Blog, Introducing the GitHub Markdown Helpers Public Beta](https://github.blog/changelog/2023-03-15-introducing-the-github-markdown-helpers-public-beta/) to learn more.
